generator/definition.go

```diff
@@ -0,0 +1,228 @@
+package generator
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/go-swagno/swagno/components/definition"
+	"github.com/go-swagno/swagno/http/response"
+	"github.com/go-swagno/swagno/utils"
+)
+
+type DefinitionGenerator struct {
```

Comment:
i like the idea of refactoring this away from the generate.go file, but now that you pulled it out I'm seeing that it's solely acting as a wrapper for `definition.Definition`, so i'm wondering if it'd be better placed in the `definition` package and letting that object generate it's own definition instead of wrapping it.

Reply:
yes, we can evaluate that